spec/support/shared_examples/dont_create_model_instance.rb

```diff
@@ -0,0 +1,5 @@
+shared_examples_for "don't create model instance" do |model|
+  it 'doesnt change count' do
+    expect{ subject }.to_not change{ model.count }
```

Comment:
@Gataniel I'm not sure if it's make sense to move this one `expcet` to shared examples... @shir what do you think?